src/views/tables/PositionsTable.tsx

```diff
@@ -359,9 +365,13 @@ export const PositionsTable = ({
   const isAccountViewOnly = useSelector(calculateIsAccountViewOnly);
   const perpetualMarkets = useSelector(getPerpetualMarkets, shallowEqual) || {};
   const assets = useSelector(getAssets, shallowEqual) || {};
-  const openPositions = useSelector(getExistingOpenPositions, shallowEqual) || [];
   const openOrders = useSelector(getSubaccountOpenOrders, shallowEqual) || [];
 
+  const openPositions = useSelector(getExistingOpenPositions, shallowEqual) || [];
+  const positions = currentMarket
+    ? openPositions.filter((position) => position.id == currentMarket)
```

Comment:
@jaredvu will we have multiple positions per market with isolated margin?

if it's single position, i think we can just do `.find`, or have a separate selector that uses current market id, so it can just have currentMarketPosition (and we don't need the currentMarket prop)

Reply:
We will only have a single position. Cannot open a `cross` position for a market with existing `isolated` position